src/screens/home/components/searcher/style.css

```diff
@@ -0,0 +1,18 @@
+.search {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    margin-left: 300px;
+}
+
+.search input {
+    background-color: #f1eded;
+    border: none;
+    border-radius: 5px;
+    padding: 8px;
+    width: 300px;
+}
+
+.search svg {
+    margin-left: 5px;
+}
```

Comment:
Finally we don't implement the search box, you must remove this entirely component